src/drizzle/0011_phoenix-schemas.sql

```diff
@@ -0,0 +1,49 @@
+CREATE EXTENSION IF NOT EXISTS citext;--> statement-breakpoint
+
+CREATE TABLE "removal_notices" (
+	"id" uuid PRIMARY KEY NOT NULL,
+	"publication_name" varchar(255),
+	"reason" text,
+	"user_id" uuid,
```

Comment:
If the `users` table becomes `admins`, I think it would be still okay to keep `user_id`.

However I think it would be clearer to have `account_id` since it links to the `account` table. Having `user` and `account` might get really confusing quickly.